lookout/style/format/postprocess.py

```diff
@@ -1,181 +1,265 @@
 """Postprocess predictions of the model."""
```

Comment:
since we create a new class and it's the only content of the file, we should rename the file `uast_stability_checker`